ddtrace/contrib/tornado/settings.py

```diff
@@ -0,0 +1,8 @@
+"""
+This module defines Tornado settings that are shared between
```

Comment:
We usually refer to `settings` when it is about configurable options (like the Django settings). We could call this `constants` instead.